requirements.txt

```diff
@@ -1,4 +1,5 @@
 ibm_cloud_sdk_core==3.11.3
+urllib3==1.26.6
```

Comment:
This itself is a dependency of `requests` - so we should not pin a version here. I think we can avoid the import entirely by using the timeout tuple. If for some reason we really do have to import it, then it shoud be a range matching whatever range the `requests` lib uses.

Reply:
Removed: 24bad01